README.md

```diff
@@ -82,19 +82,11 @@ docker run -it --rm
 
 You can see all the supported environment variables in [this document](./docs/env-variables.md).
 
-## Unsupported Features
-
 ### Development Containers
 
-The table below keeps track of features we plan to implement. Feel free to [create a new issue](https://github.com/coder/envbuilder/issues/new) if you'd like Envbuilder to support a particular feature.
+[This document](./docs/supported-features.md) keeps track of what parts of the Dev Container specification Envbuilder currently supports.
```

Comment:
We need to change it to dev-container-spec-support.md

Reply:
whoops!